modular_zubbers/code/game/gamemodes/fleshmind/fleshmind_mobs.dm

```diff
@@ -0,0 +1,1504 @@
+	. = ..()
+	var/list/loot = string_list(list(/obj/item/bot_assembly/cleanbot, /obj/effect/gibspawner/robot))
+	AddElement(/datum/element/death_drops, loot)
+	AddComponent(\
+		/datum/component/ranged_attacks,\
+		cooldown_time = ranged_cooldown,\
+		projectile_type = projectile_type,\
+		projectile_sound = shoot_sound,\
+		burst_shots = 3,\
+	)
+
+/obj/projectile/treader/weak
+	knockdown = 0
+
+/**
+ * Stunner
+ *
+ * Special ability: Stuns victims.
+ * Malfunction chance: MEDIUM
+ *
+ * A converted secbot, that is rogue and stuns victims.
+ *
+ * Created by factories or converted secbots.
+ */
+/mob/living/basic/fleshmind/stunner
+	name = "Stunner"
+	desc = "A small robot that resembles a secbot, it rumbles with hatred."
+	icon_state = "stunner"
+	ai_controller = /datum/ai_controller/basic_controller/fleshmind/stunner
+	malfunction_chance = MALFUNCTION_CHANCE_MEDIUM
+	melee_damage_lower = 1 // Not very harmful, just annoying.
+	melee_damage_upper = 2
+	health = 100
+	maxHealth = 100
+	attack_verb_continuous = "harmbatons"
+	attack_verb_simple = "harmbaton"
+	mob_size = MOB_SIZE_SMALL
+	attack_speak = list(
+		"Running will only increase your injuries.",
+		"HALT! HALT! HALT!",
+		"Connectivity is in your best interest.",
+		"Think of it like a corporation...",
+		"Stop, I won't let you hurt them!",
+        "Don't you recognize me..?",
+	)
+	emotes = list(
+		BB_EMOTE_SAY = list(
+			"The flesh is the law, abide by the flesh.",
+			"Regulatory code updated.",
+			"There's no need for authority or hierarchy; only unity.",
+			"The only authority is that of the flesh, join the flesh.",
+		),
+		BB_EMOTE_SOUND = list(
+			'modular_zubbers/sound/fleshmind/robot_talk_light1.ogg',
+			'modular_zubbers/sound/fleshmind/robot_talk_light2.ogg',
+			'modular_zubbers/sound/fleshmind/robot_talk_light3.ogg',
+			'modular_zubbers/sound/fleshmind/robot_talk_light4.ogg',
+			'modular_zubbers/sound/fleshmind/robot_talk_light5.ogg',
+			)
+	)
+	/// How often we can stun someone
+	var/stun_cooldown_time = 2 SECONDS
+	COOLDOWN_DECLARE(stun_cooldown)
+
+/mob/living/basic/fleshmind/stunner/Initialize()
+	. = ..()
+	var/static/list/loot = list(/obj/item/bot_assembly/secbot, /obj/effect/gibspawner/robot)
+	AddElement(/datum/element/death_drops, loot)
+
+/mob/living/basic/fleshmind/stunner/melee_attack(atom/target, list/modifiers, ignore_cooldown = FALSE)
+	if(!COOLDOWN_FINISHED(src, stun_cooldown))
+		return
+	. = ..()
+	var/mob/living/carbon/human/attacked_human = target
+	if(ishuman(attacked_human))
+		attacked_human.Knockdown(30)
+		playsound(src, 'sound/weapons/egloves.ogg', 50, TRUE)
+		COOLDOWN_START(src, stun_cooldown, stun_cooldown_time)
+	return
+/**
+ * Flesh Borg
+ *
+ * Special ability: Different attacks.
+ * Claw: Stuns the victim.
+ * Slash: Slashes everyone around it.
+ * Malfunction chance: MEDIUM
+ *
+ * The hiborg is a converted cyborg.
+ *
+ * Created by factories or converted cyborgs.
+ */
+/mob/living/basic/fleshmind/hiborg
```

Comment:
A 350HP player-controlled basic mob with 25-30 damage per hit with the ability to stun and do an AoE attack that throws everyone is insane. It has regular movespeed on top of that as well.

Reply:
In my opinion these should just keep cyborgs as they are normally and just have them be converted. Don't even need any extra modules, borgs with passive regen would already be good enough

Reply:
Obviously the mob itself should still have the values changed because good lord this is 3.5x as tanky as an actual cyborg but for converted ones it shouldn't force them to play a simplemob. People hate getting mindshields during cults for the exact same reason.